src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/DiagnosticsOverlay.xaml

```diff
@@ -269,14 +270,21 @@
 				<RowDefinition Height="*" />
 			</Grid.RowDefinitions>
 
-			<StackPanel DataContext="{Binding Counters}"
-						HorizontalAlignment="Right"
-						Padding="4"
-						xamarin:IsHitTestVisible="False">
+			<Grid HorizontalAlignment="Right"
```

Comment:
Why add a new grid?
Also, not all child elements are aligned with that new addition.